cmd/notary/tuf.go

```diff
@@ -115,6 +117,7 @@ func (t *tufCommander) AddToCommand(cmd *cobra.Command) {
 
 	cmdTUFAdd := cmdTUFAddTemplate.ToCommand(t.tufAdd)
 	cmdTUFAdd.Flags().StringSliceVarP(&t.roles, "roles", "r", nil, "Delegation roles to add this target to")
+	cmdTUFAdd.Flags().BoolVarP(&t.autoPublish, "publish", "p", false, "Automatically attempt to publish after adding the change. Will also publish those pre-generated changelists.")
```

Comment:
nit on the phrasing: `"Automatically attempt to publish after staging the change. Will also publish existing staged changes."`